website/app/components/doc/page/header/search/index.js

```diff
@@ -0,0 +1,207 @@
+              },
+            },
+          ];
+        } else {
+          return [
+            {
+              sourceId: 'suggestions',
+              getItems() {
+                return [
+                  {
+                    type: 'suggestion',
+                    pageTitle: 'Support: How to file an issue',
+                    searchResultURL: '/about/support#file-an-issue',
+                    previewIcon: 'support',
+                  },
+                  {
+                    type: 'suggestion',
+                    pageTitle: 'What‘s new: Release notes',
+                    searchResultURL: '/whats-new/release-notes',
+                    previewIcon: 'newspaper',
+                  },
+                  {
+                    type: 'suggestion',
+                    pageTitle: 'Icons: Library',
+                    searchResultURL: '/icons/library',
+                    previewIcon: 'search',
+                  },
+                ];
+              },
+              getItemUrl({ item }) {
+                return item.searchResultURL;
+              },
+              templates: {
+                header: templatesHeaderFunction({ group: 'suggestions' }),
+                // footer: templatesFooterFunction,
+                item: templatesItemFunction(),
+              },
+            },
+          ];
+        }
+      },
+      render({ elements, render, html }, root) {
+        const { suggestions, content, icons, tokens, recentSearchesPlugin } =
+          elements;
+
+        render(
+          html`
+            <div class="aa-PanelLayout">
+              <div class="aa-PanelSources">
+                <!-- SUGGESTIONS -->
+                ${suggestions}
+                <!-- RECENT SEARCHES -->
+                <!-- RECENT SEARCHES -->
+                <!--${recentSearchesPlugin}</div>-->
+                <!-- CONTENT ("GENERIC") -->
+                ${content}
+                <!-- ICONS -->
+                ${icons}
+                <!-- TOKENS -->
+                ${tokens}
+              </div>
+            </div>
+            ${htmlPanelFooter({ html })}
+          `,
+          root
+        );
+      },
+      renderNoResults({ render, html, state }, root) {
+        render(
+          html`
+            <div class="aa-PanelLayout aa-Panel--scrollable">
+              ${htmlPanelNoResults({ html, state })}
+            </div>
+            ${htmlPanelFooter({ html })}
+          `,
+          root
+        );
+      },
+    });
+
+    const alternativeTrigger = document.getElementById(
+      'doc-algolia-search-autocomplete-secondary-trigger'
+    );
+    alternativeTrigger.addEventListener('click', () => {
+      console.log('THIS SHOULD OPEN THE SEARCH');
+      autocompleteInstance.setIsOpen(true);
+    });
+
+    // add event listener to open/close the modal via `command-K` hot-key
+    document.addEventListener('keydown', (event) => {
+      if (event.metaKey && event.key === 'k') {
```

Comment:
I'll finalize the command to use here later (https://hashicorp.atlassian.net/browse/HDS-2879)